src/main/java/com/yelp/nrtsearch/server/luceneserver/ServerCodec.java

```diff
@@ -21,13 +21,13 @@
 import org.apache.lucene.codecs.Codec;
 import org.apache.lucene.codecs.DocValuesFormat;
 import org.apache.lucene.codecs.PostingsFormat;
-import org.apache.lucene.codecs.lucene84.Lucene84Codec;
+import org.apache.lucene.codecs.lucene95.Lucene95Codec;
 
 /** Implements per-index {@link Codec}. */
-public class ServerCodec extends Lucene84Codec {
+public class ServerCodec extends Lucene95Codec {
```

Comment:
Do we want to maintain compatibility with Lucene 8 indices? https://github.com/apache/lucene/blob/main/lucene/backward-codecs/src/java/org/apache/lucene/backward_codecs/lucene84/Lucene84Codec.java

Can also add later if needed.

Reply:
This is something that will require some testing to see what is needed to fully support a v8 index. This can be evaluated later as needed.